scripts/ci/run_combine.sh

```diff
@@ -1,33 +1,41 @@
 #!/bin/bash
 
-if [[ $# -lt 2 ]]; then
-	echo "Requires at least two arguments: run_combine.sh <combinetf_dir> <combine_cards>"
+if [[ $# -lt 4 ]]; then
+	echo "Requires at least four arguments: run_combine.sh <combinetf_dir> <mode> <working_dir> <combine_cards>"
 	exit 1
 fi
 
 combinetf_dir=$1
+mode=$2
+working_dir=$3
 
 cards=()
-for card in ${@:2}; do
+for card in ${@:4}; do
 	echo $card
-	cards+=( `basename $card` )
+	cards+=( $card )
 done
 
 source /cvmfs/cms.cern.ch/cmsset_default.sh
 pushd $combinetf_dir
 eval `scram runtime -sh`
 popd
 
-working_dir=`dirname $2`
-card_name=${cards[0]}
-
 pushd $working_dir
 
-if [[ $# -gt 2 ]]; then
-	card_name=${working_dir}.txt
-	combineCards.py $cards > $card_name
-fi
+card_name=${working_dir}.txt
+echo "combineCards.py ${cards[@]} > $card_name"
+combineCards.py ${cards[@]} > $card_name
 
 outfile=${card_name/txt/hdf5}
-text2hdf5.py --X-allow-no-signal $card_name
-combinetf.py --doImpacts --binByBinStat -t -1 $outfile
+
+if [ "$mode" == "mass" ]; then
+	echo text2hdf5.py --X-allow-no-signal "$card_name"
```

Comment:
You can run with bash +x and it will output every command it runs rather than calling echo for every command

Reply:
Thanks, didn't know that! Done